### ISSUE TITLE
The macro allows `$ty` to be followed `...` which will be invalid soon

### ISSUE THREAD

**Issue by cristicbz** [vberger/dlib#4]
```
<dlib macros>:6:59: 6:62 warning: `$vargs:ty` may be followed by `...`, which is not allowed for `ty` fragments
<dlib macros>:6 varargs : $ ( fn $ vname : ident ( $ ( $ vargs : ty ) , + ... ) -> $ vret : ty
                                                                          ^~~
<dlib macros>:6:59: 6:62 note: The above warning will be a hard error in the next release.
<dlib macros>:6 varargs : $ ( fn $ vname : ident ( $ ( $ vargs : ty ) , + ... ) -> $ vret : ty
```

Not sure what the solution is; maybe change the syntax to be `c_int, c_int, [...]` or something like that.

**Comment by vberger** [comment]
This is already fixed in `v0.3.0`, that's been on crates.io for two months.

If you depend on dlib through wayland-client, you'll probably just need to `cargo update`

**Comment by cristicbz** [comment]
Sorry about that! I depend on `wayland-client` via a long chain of deps; will trace down who's at fault, thanks!
